modules/gitops-release-notes-1-4-12.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-4-12_{context}"]
+= Release notes for {gitops-title} 1.4.12
+
+{gitops-title} 1.4.12 is now available on {product-title} 4.8, 4.9, and 4.10.
+
+[id="fixed-issues-1-4-12_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* Before this update, in a large set of application the application controllers were restarted multiple times due to the unresponsiveness of liveness probes. This update fixes the issue by removing the liveness probe in the application controller `statefulset`.  link:https://issues.redhat.com/browse/GITOPS-2153[GITOPS-2153]
+
+* Before this update, the certificate of RHSSO cannot be validated when it is setup with a certificate which is not signed by certificate authorities. This update fixes the issue and now the users can provide a custom certificate which will be used in verifying the Keycloak's TLS certificate when communicating with it. link:https://issues.redhat.com/browse/GITOPS-2214[GITOPS-2214]
+
```

Comment:
please include the details and instructions outlined in the  Release notes text of  https://issues.redhat.com/browse/GITOPS-2214